source/DasBlog.Web.Repositories/Interfaces/ISiteSecurityRepository.cs

```diff
@@ -5,38 +5,38 @@
 
 namespace DasBlog.Web.Repositories.Interfaces
 {
-    public interface ISiteSecurityRepository
-    {
-        bool IsValidContributor { get; }
+	public interface ISiteSecurityRepository
+	{
+		bool IsValidContributor { get; }
 
-        bool IsInRole(string role);
+		bool IsInRole(Role? role);
```

Comment:
What does making this type nullable get us here?

Reply:
The existing logic was passing in null. It didn't look right but wasn't part of getting asp.net identity to "work" so I made it nullable. What we really should do is get rid of this and use the method attributes `[IsInRole]`